vm-migration/src/protocol.rs

```diff
@@ -139,19 +134,14 @@ impl Request {
 }
 
 #[repr(u16)]
-#[derive(Copy, Clone, PartialEq, Eq)]
+#[derive(Copy, Clone, PartialEq, Eq, Default)]
 pub enum Status {
+    #[default]
     Invalid,
     Ok,
     Error,
 }
 
-impl Default for Status {
-    fn default() -> Self {
```

Comment:
I'm surprised clippy only flagged this. We have much more of these kind of blocks in our code